enctests/testframework/main.py

```diff
@@ -407,10 +475,17 @@ def idiff_compare(source_clip, test_ref, testname, comparisontestinfo, source_pa
     if cmdresult != 0:
         result['result'] = "Unable to extract file for test"
     else:
-        cmd = apptemplate.format(originalfile=sourcepng, newfile=distortedpng, idiff_bin=IDIFF_BIN, newfilediff=diffpng)
+        cmd = apptemplate.format(originalfile=sourcepng, 
+                                 newfile=distortedpng.as_posix(),
+                                idiff_bin=IDIFF_BIN, 
+                                newfilediff=diffpng.as_posix())
         print(f"\n\nIdiff command: {cmd}", file=log_file_object)
         
-        output = subprocess.run(shlex.split(cmd), check=False, stdout=subprocess.PIPE).stdout
+        try:
+            output = subprocess.run(shlex.split(cmd), check=False, stdout=subprocess.PIPE).stdout
+        except FileNotFoundError as e:
+            print(f"ERROR: File not found, when executing {cmd}, check that the executable exists.")
+            exit(1)
         lines = output.decode("utf-8").splitlines()
         print("\n".join(lines), file=log_file_object)
         if len(lines) < 2:
```

Comment:
Just mentioning this if we go the return code route mentioned above

Reply:
As we mentioned further up, not finding ffmpeg feels pretty fatal :(